### ISSUE TITLE
Implement the All Contributors Specification

### ISSUE THREAD

**Issue by KagemniKarimu** [chainbase-labs/manuscript-core#77]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/chainbase-labs/manuscript-core/issues), including closed ones.
- [X] I have searched for existing pull requests [search for existing pull requests](https://github.com/chainbase-labs/manuscript-core/pulls), including closed ones.

### Provide a description of requested docs changes

# Recommendation
:star: All Contributors voices are valued. We should implement the All Contributors spec to give shine to people who have given themselves in other ways besides code: https://allcontributors.org/docs/en/overview

From the website:
> People are giving themselves and their free time to contribute to open source projects in so many ways, so we believe everyone should be praised for their contributions (code or not). Projects should define contributors as those who contribute to a project in any of the Contribution Categories at any level of contribution. This specification is, by definition, inclusive of all contributions. 

# Implementation

The easiest way to do it is via the All Contributors bot.
https://allcontributors.org/docs/en/bot/installation

## Broad steps

1. [Install the GitHub App](https://github.com/apps/allcontributors/installations/new)
2. Allow Permissions
3. Edit `README.md` to fit bot parsing :
```markdown
## Contributors

<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
<!-- prettier-ignore-start -->
<!-- markdownlint-disable -->

<!-- markdownlint-restore -->
<!-- prettier-ignore-end -->

<!-- ALL-CONTRIBUTORS-LIST:END -->
```
4. Add Badge to `README.md`:

```markdown
[![All Contributors](https://img.shields.io/github/all-contributors/projectOwner/projectName?color=ee8449&style=flat-square)](#contributors)
```
(Adjust color & style to match other shields)

5. Create a new file: `.all-contributorsrc` , optionally importing current contributors. Necessary for the bot to function.
```json
{
  "projectName": "<insert the repo's name>",
  "projectOwner": "<insert the repo's owner/orgs>"
}
```

6. Update our `Contributor.md` to explain how the bot works/how contributions are counted.

**Comment by Liquidwe** [comment]
Yes, we take Contributors very seriously, and the repository currently has a simple version of bot：https://github.com/chainbase-labs/manuscript-core/blob/main/.github/workflows/contribute.yml
